public/app/core/services/bridge_srv.ts

```diff
@@ -84,7 +83,7 @@ export class BridgeSrv {
         if (changes) {
           const dash = getDashboardSrv().getCurrent();
           if (dash) {
-            this.variableSrv.templateVarsChangedInUrl(changes);
```

Comment:
@ryantxu if you have time to test that this doesn't break the previous use case I would really appreciate it!

Reply:
thanks for testing this out @ryantxu